ui/component/claimPreview/view.jsx

```diff
@@ -175,6 +175,10 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
     );
   }
 
+  if (placeholder === 'publish' && !claim && uri.startsWith('lbry://@')) {
+    return null;
```

Comment:
We could still render the preview and open the channel create modal when they click on it.